### PR TITLE
Do a random walk, come out with random mappings

### PR DESCRIPTION
**Pull request by EnnuiL**
This establishes a foundation on some random areas like the new ArgumentType and container stuff, but unfortunately not to a point where they're complete

### CODE REVIEW

File: mappings/net/minecraft/client/gui/screen/ConfirmLinkScreen.mapping [hunk 1]
```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/unmapped/C_vpzsgabs net/minecraft/client/gui/screen/ConfirmChatLinkScreen
+CLASS net/minecraft/unmapped/C_vpzsgabs net/minecraft/client/gui/screen/ConfirmLinkScreen
```
Comment (eristhea) [inline mappings/net/minecraft/client/gui/screen/ConfirmLinkScreen.mapping:1]: what's the logic behind this change?

**Review by OroArmor — APPROVED** [review]

File: mappings/net/minecraft/command/argument/BlockPredicateArgumentType.mapping [hunk 2]
```diff
@@ -1,5 +1,5 @@
 CLASS net/minecraft/unmapped/C_oicjkagd net/minecraft/command/argument/BlockPredicateArgumentType
-	FIELD f_fvydyaji blocks Lnet/minecraft/unmapped/C_vtbxyypo;
+	FIELD f_fvydyaji holderLookup Lnet/minecraft/unmapped/C_vtbxyypo;
```
Comment (eristhea) [inline mappings/net/minecraft/command/argument/BlockPredicateArgumentType.mapping:2]: again, what's the logic here?

File: mappings/net/minecraft/command/argument/DimensionArgumentType.mapping [hunk 3]
```diff
@@ -6,7 +6,7 @@ CLASS net/minecraft/unmapped/C_agjrdwvm net/minecraft/command/argument/Dimension
 		ARG 1 context
 		ARG 2 builder
 	METHOD m_nscsmxxv (Ljava/lang/Object;)Lcom/mojang/brigadier/Message;
-		ARG 0 id
+		ARG 0 dimension
```
Comment (eristhea) [inline mappings/net/minecraft/command/argument/DimensionArgumentType.mapping:9]: I don't like this, what's being passed isn't a dimension object and the fact that it's a dimension id is easily inferred

File: mappings/net/minecraft/command/argument/ParticleEffectArgumentType.mapping [hunk 4]
```diff
@@ -1,18 +1,28 @@
...
+	FIELD f_ebwimnfp holderLookup Lnet/minecraft/unmapped/C_vtbxyypo;
+	METHOD <init> (Lnet/minecraft/unmapped/C_bprdujxm;)V
+		ARG 1 context
 	METHOD getExamples getExamples ()Ljava/util/Collection;
 	METHOD listSuggestions listSuggestions (Lcom/mojang/brigadier/context/CommandContext;Lcom/mojang/brigadier/suggestion/SuggestionsBuilder;)Ljava/util/concurrent/CompletableFuture;
 		ARG 1 context
 		ARG 2 builder
+	METHOD m_frlhxnkn (Lcom/mojang/brigadier/StringReader;Lnet/minecraft/unmapped/C_vtbxyypo;)Lnet/minecraft/unmapped/C_qlzoaopu;
+		ARG 0 reader
+		ARG 1 holderLookup
```
Comment (eristhea) [inline mappings/net/minecraft/command/argument/ParticleEffectArgumentType.mapping:13]: again, I think it's better to name these holder lookups based on what they're providing than simply holder lookup

File: mappings/net/minecraft/command/argument/ParticleEffectArgumentType.mapping [hunk 5]
```diff
@@ -1,18 +1,28 @@
...
 	METHOD m_jksfcpto readParameters (Lcom/mojang/brigadier/StringReader;Lnet/minecraft/unmapped/C_qlzoaopu;)Lnet/minecraft/unmapped/C_nqucohct;
 		ARG 0 reader
 		ARG 1 type
 	METHOD m_jytmlznx particleEffect (Lnet/minecraft/unmapped/C_bprdujxm;)Lnet/minecraft/unmapped/C_wvjakmzf;
+		ARG 0 context
 	METHOD m_sdajnwtg (Ljava/lang/Object;)Lcom/mojang/brigadier/Message;
-		ARG 0 id
+		ARG 0 particleEffect
```
Comment (eristhea) [inline mappings/net/minecraft/command/argument/ParticleEffectArgumentType.mapping:26]: questionable id name change #2

**Review by eristhea — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/client/gui/screen/ConfirmLinkScreen.mapping (hunk 1 above)
Comment (EnnuiL) [reply mappings/net/minecraft/client/gui/screen/ConfirmLinkScreen.mapping:1]: It isn't used only for chat links; Yes, it was it's primary focus, but I've noticed that it became only one of its possible usages; Good example: the telemetry data screen and how it uses the screen for the feedback link

Comment (eristhea) [reply mappings/net/minecraft/client/gui/screen/ConfirmLinkScreen.mapping:1]: good change then :)

File: mappings/net/minecraft/command/argument/BlockPredicateArgumentType.mapping (hunk 2 above)
Comment (EnnuiL) [reply mappings/net/minecraft/command/argument/BlockPredicateArgumentType.mapping:2]: As far as I've seen, `blocks` isn't a standard at all, and I suspect that it was a leftover from pre-1.19.3 registry changes, so I've changed it to `holderLookup` for greater consistency; Maybe there is a better alternative? But I don't think `blocks` is one

File: mappings/net/minecraft/command/argument/DimensionArgumentType.mapping (hunk 3 above)
Comment (EnnuiL) [reply mappings/net/minecraft/command/argument/DimensionArgumentType.mapping:9]: No, it isn't an id either, it's the DimensionArgumentTy- wait, did enigma trick me

File: mappings/net/minecraft/command/argument/BlockPredicateArgumentType.mapping (hunk 2 above)
Comment (eristhea) [reply mappings/net/minecraft/command/argument/BlockPredicateArgumentType.mapping:2]: sure, if it's in the name of consistency I don't mind

File: mappings/net/minecraft/command/argument/DimensionArgumentType.mapping (hunk 3 above)
Comment (eristhea) [reply mappings/net/minecraft/command/argument/DimensionArgumentType.mapping:9]: oh wait am I dumb

Comment (eristhea) [reply mappings/net/minecraft/command/argument/DimensionArgumentType.mapping:9]: I think you're right here; that is indeed the dimension being passed

**Review by eristhea — APPROVED** [review]

File: mappings/net/minecraft/command/argument/DimensionArgumentType.mapping (hunk 3 above)
Comment (EnnuiL) [reply mappings/net/minecraft/command/argument/DimensionArgumentType.mapping:9]: no i was wrong, it was the dimension ID; although yeah, this does point to a bad standard going on

File: mappings/net/minecraft/registry/HolderLookup.mapping [hunk 6]
```diff
@@ -1,6 +1,6 @@
 CLASS net/minecraft/unmapped/C_vtbxyypo net/minecraft/registry/HolderLookup
 	METHOD m_cjhuibvn holders ()Ljava/util/stream/Stream;
-	METHOD m_pgmbmohw steamTagKeys ()Ljava/util/stream/Stream;
+	METHOD m_pgmbmohw streamTagKeys ()Ljava/util/stream/Stream;
```
Comment (LambdAurora) [inline mappings/net/minecraft/registry/HolderLookup.mapping:3]: Wasn't I the one who mapped this?

oops

**Review by LambdAurora — APPROVED** [review]

**Pull request merged**